synapse/storage/databases/main/room.py

```diff
@@ -1320,6 +1320,58 @@ async def add_event_report(
             desc="add_event_report",
         )
 
+    async def get_event_reports_paginate(
+        self, start: int, limit: int, user_id: str = None, room_id: str = None
+    ) -> Tuple[List[Dict[str, Any]], int]:
+        """Function to retrieve a paginated list of event reports
+        This will return a json list of event reports and the
+        total number of event reports matching the filter criteria.
+
+        Args:
+            start: start number to begin the query from
+            limit: number of rows to retrieve
+            user_id: search for user_id. ignored if name is not None
+            room_id: search for room_id. ignored if name is not None
+        """
+
+        def _get_event_reports_paginate_txn(txn):
+            filters = []
+            args = []
+
+            if user_id:
+                filters.append("user_id LIKE ?")
+                args.extend(["%" + user_id + "%"])
+            if room_id:
+                filters.append("room_id LIKE ?")
+                args.extend(["%" + room_id + "%"])
+
+            where_clause = "WHERE " + " AND ".join(filters) if len(filters) > 0 else ""
+
+            sql_base = """
+                FROM event_reports
+                {}
+                """.format(
+                where_clause
+            )
+            logger.warning("sql_base: %s ", sql_base)
```

Comment:
Why warn the homeserver operator about this?

Reply:
I forgot to remove it after a debugging session.